src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,66 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final byte buySum = 0;
+    private static final byte supplySum = 1;
+    private static final String supplyOperation = "supply";
+    private static final String buyOperation = "buy";
+    private static final String result = "result";
```

Comment:
if you create constants they should be in uppercase
also constants with the same names as the value dont give any better understanding

Reply:
Ok, I'll try to change it...